.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           python-version: "3.11"
           cache: 'poetry'
       - run: pip install tox
-      - run: tox -e lint,py311-dj42
+      - run: tox -e lint,py311-dj50
```

Comment:
nitpick: `test_compatibility` below stats "Test with all supported Django versions, for all compatible Python versions." and includes 4.2 and main. Would be nice to update it to include 5.0, but that is not a blocker